src/Pages/OATEditorPage/Internal/OATErrorHandlingWrapper.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { PrimaryButton, Modal } from '@fluentui/react';
+import { useTranslation } from 'react-i18next';
+import { SET_OAT_ERROR } from '../../../Models/Constants/ActionTypes';
+import { getEditorPageStyles } from '../OATEditorPage.Styles';
+import { IOATEditorState } from '../OATEditorPage.types';
+import { IAction } from '../../../Models/Constants/Interfaces';
+
+interface IOATErrorHandlingWrapper {
+    dispatch?: React.Dispatch<React.SetStateAction<IAction>>;
+    state?: IOATEditorState;
+}
+
+const OATErrorHandlingWrapper = ({
+    state,
+    dispatch
+}: IOATErrorHandlingWrapper) => {
+    const { t } = useTranslation();
+    const EditorPageStyles = getEditorPageStyles();
```

Comment:
Should this be `editorPageStyles`? I.e. lower case `E`. Depends on the pattern that's used elsewhere.